planner/core/plan.go

```diff
@@ -377,6 +381,15 @@ func (p *baseLogicalPlan) ExplainInfo() string {
 	return ""
 }
 
+// buildLogicalPlanTrace implements LogicalPlan
+func (p *baseLogicalPlan) buildLogicalPlanTrace() tracing.LogicalPlanTrace {
+	planTrace := tracing.LogicalPlanTrace{ID: p.ID(), TP: p.TP()}
```

Comment:
explainInfo will be filled next?

Reply:
yes, `explainInfo` will be implemented by specific logical operator.